com.microsoft.mrtk.uxcore/Button/RectTransformColliderFitter.cs

```diff
@@ -203,6 +238,10 @@ private void Fit()
                         }
                     }
                 }
+                else if (!ApplyMask && thisCollider != null && !thisCollider.enabled)
+                {
+                    thisCollider.enabled = true;
```

Comment:
Why is this necessary? Colliders shouldn't be disabled in the first place.

Reply:
This is necessary to correctly support changes during runtime, in this scenario (albeit a rare case presumably):
1. Mask is enabled. Collider is disabled because it doesn't intersect the mask.
2. `UseMask` is toggled (either by editor GUI or script)
3. The collider needs to be re-enabled during the next `Fit()` as the mask should no longer apply.

Reply:
Ah, gotcha.